src/vlogger/vlogger.cpp

```diff
@@ -75,9 +75,13 @@ namespace log_level
 	static const char *log_names_warn[]    = {"warn",  "warning", "2", NULL};
 	static const char *log_names_info[]    = {"info",  "information", "3", NULL};
 	static const char *log_names_details[] = {"details", NULL};
+#if !defined(VMA_OPTIMIZE_LOG) || (VMA_OPTIMIZE_LOG > 5)
 	static const char *log_names_debug[]   = {"debug", "4", NULL};
+#endif
+#if !defined(VMA_OPTIMIZE_LOG) || (VMA_OPTIMIZE_LOG > 6)
```

Comment:
It uses the same logic as exists now in this case. Now VMA sets default level if sees invalid argument and reports it in the banner as
`VMA INFO: Log Level                      INFO                       [VMA_TRACELEVEL]`